ttnn/cpp/ttnn/operations/moreh/moreh_clip_grad_norm/moreh_clip_grad_norm_step1/device/kernels/moreh_clip_grad_norm_step1_kernel.cpp

```diff
@@ -58,9 +58,8 @@ void MAIN {
     for (uint32_t tile_idx = 0; tile_idx < num_tiles; tile_idx++) {
         // Comput cb_xabs and mask(optional)
         // |x|
-        ACQ();
+        tile_regs_acquire();
         cb_wait_front(cb_x, onetile);  // comes from the reader
-        cb_reserve_back(cb_xabs, onetile);
```

Comment:
You can keep reserve_back/push_back and wait_front/pop_front at the start/end of each computation step to keep track better of which cb you're reading/writing.

Reply:
resolved in [ce8237a](https://github.com/tenstorrent/tt-metal/pull/15897/commits/ce8237a7c45fd186600377fcd4f57975e5e5fd4e)